docs/kyma/03-01-components.md

```diff
@@ -45,6 +43,30 @@ Kyma comes bundled with tools that give you the most accurate and up-to-date mon
 
 The tracing in Kyma uses the [Jaeger](https://github.com/jaegertracing) distributed tracing system. Use it to analyze performance by scrutinizing the path of the requests sent to and from your service. This information helps you optimize the latency and performance of your solution.
 
+## API Gateway
+
+The API Gateway aims to provide a set of functionalities which allow developers to expose, secure, and manage their API's in an easy way. The main elemennt of the API Gateway is the API Controller which exposes services in Kyma.
+
 ## Logging
 
 Logging in Kyma uses [Loki](https://github.com/grafana/loki), a Prometheus-like log management system.
+
+## Backup
+
+Kyma integrates with [Velero](https://github.com/heptio/velero/) to provide backup and restore capabilities for Kubernetes cluster resources. Once backed up, Velero stores the resources in buckets of [supported cloud providers](https://velero.io/docs/v1.0.0/support-matrix/).
+
+## Console
+
+The Console is a web-based administrative UI for Kyma. It uses the [Luigi framework](https://github.com/SAP/luigi) to allow you to seamlessly extend the UI content with custom micro frontends. A backend service which provides an API for all views of the Console UI is the Console Backend Service.
+
+## Asset Store
+
+The Asset Store is a flexible, scalable, multi-cloud, and location-independent Kubernetes-native solution for storing assets, such as documents, files, images, API specifications, and client-side applications. Its main components are Asset and Bucket custom resources (CRs), Asset and Bucket Controllers, and Minio Gateway as a default storage solution. The Asset Store also allows you to use optional services, such as Validation, Mutation, and Metadata Services.
+
+## Headless CMS
+
+The Headless CMS is a new kind of Content Management Systems (CMS) that provides a way of storing and managing raw content, and exposing it through an API. It only ensures a database backend without providing a display layer. In Kyma, the Headless CMS allows you to upload multiple and grouped data for a given documentation topic and store them as Asset CRs in Minio buckets.
```

Comment:
`is a new kind of Content Management Systems (CMS)` -> `is a Kubernetes native Content Management Systems (CMS)`

please rephrase 
```
It only ensures a database backend without providing a display layer. In Kyma, the Headless CMS allows you to upload multiple and grouped data for a given documentation topic and store them as Asset CRs in Minio buckets.
```

just write that hcms is an abstraction on top of AssetStore and this way, out of the box provides only a data layer. There is no separate dedicated presentation layer but it is integrated with Console UI that consumes content stored with hcms